.gitignore

```diff
@@ -49,6 +49,11 @@
 !/media/dhl/logo.jpg
 !/media/downloadable/.htaccess
 
+/shell/*
```

Comment:
These lines would actually break my customer’s projects cause we have a lot of scripts in that forlder

Reply:
yeah, In nearly every project I worked there were scripts put in there.